OtripleS.Portal.Web/.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I guess this was not intentionally added

Reply:
@coeur85 same thing let's remove this file as well please.

Reply:
done